osconfig_tests/compute/instance.go

```diff
@@ -82,6 +89,50 @@ func (i *Instance) WaitForSerialOutput(match string, port int64, interval, timeo
 	}
 }
 
+// StreamSerialOutput stores the serial output of an instance to GCS bucket
+func (i *Instance) StreamSerialOutput(ctx context.Context, storageClient *storage.Client, logsPath, bucket string, logwg *sync.WaitGroup, port int64, interval time.Duration) {
+	defer logwg.Done()
+
+	logsObj := path.Join(logsPath, fmt.Sprintf("%s-serial-port%d.log", i.Name, port))
+	logger.Infof("Streaming instance %q serial port %d output to https://storage.cloud.google.com/%s/%s", i.Name, port, bucket, logsObj)
+	var start int64
+	var buf bytes.Buffer
+	var isTerminalStatus bool
```

Comment:
remove `isTerminalStatus` since it's not used anymore.

Reply:
done